lib/worker/init.js

```diff
@@ -27,8 +28,9 @@ if (self.location.protocol === 'file:') {
   global.module.filename = global.__filename
 
   // Also search for module under the html file.
-  global.module.paths = global.module.paths.concat(Module._nodeModulePaths(global.__dirname))
+  global.module.paths = Module._nodeModulePaths(global.__dirname)
 } else {
-  global.__filename = __filename
-  global.__dirname = __dirname
+  // For backwards compatibility we fake these two paths here
```

Comment:
what depends on this?

Reply:
User code, `__dirname` and `__filename` are globals in node land that are expected to exist and be valid.  When the internal `init` doesn't exist on disk node doesn't set these values so here I fake them to their old values.  In all likelihood we will have to keep this logic forever 👍

Reply:
wouldn't those variables be in relation to the user's files though, not the asar?